code/game/mecha/mech_fabricator.dm

```diff
@@ -461,6 +461,19 @@
 								"Pod_Parts",
 								"Pod_Frame",
 								"Misc")
+	req_access = list(access_mechanic)
+
+/obj/machinery/mecha_part_fabricator/spacepod/New()
+	. = ..()

```

Comment:
Why do you need to return the parent's value?

Reply:
I don't. Did that out of habit. Changed now.